commands/crit.js

```diff
@@ -137,7 +146,7 @@ module.exports = {
 		}
 		return await ctx.send(icon1 + icon2, getEmbedCrit(crit, fileName, ctx, lang))
 			.then(() => {
-				if (crit.fatal) {
+				if (crit.fatal && game !== 'vaesen') {
```

Comment:
Why not use crit.game here, too?

Reply:
No real reason. `game` and `crit.game` have the same value, so I chose the shortest variable name.

Reply:
Ah, alright. Makes no difference anyway, just personal preference i suppose.

Reply:
You are right, `crit.game` feels better to me too.

But I have future plans to revamp the crit commands because of how the Twilight 2000 critical injuries work, and I am not sure to keep `crit.game`.